docs/references/glossary.md

```diff
@@ -0,0 +1,328 @@
+
+An organization's portfolio of software products managed in Backstage.
+
+Can also be: The core Backstage plugin that handle ingestion and display of your organizations software products.
+
+## Cloud Native Computing
+
+A set of technologies that "empower organizations to build and run scalable applications in modern, dynamic environments such as public, private, and hybrid clouds. Containers, service meshes, microservices, immutable infrastructure, and declarative APIs exemplify this approach." ([CNCF Cloud Native Definition v1.0](https://github.com/cncf/toc/blob/main/DEFINITION.md)).
+
+## Cloud Native Computing Foundation
+
+A foundation dedicated to the promotion and advancement of [Cloud Native Computing](#Cloud-Native-Computing). The mission of the Cloud Native Computing Foundation (CNCF) is "to make cloud native computing ubiquitous" ([CNCF Charter](https://github.com/cncf/foundation/blob/main/charter.md)).
+
+CNCF is part of the [Linux Foundation](https://www.linuxfoundation.org/).
+
+## CNCF
+
+Cloud Native Computing Foundation.
+
+## Code Grant
+
+[OAuth](#oauth) flow where the client receives an [authorization code](#code) that is passed to the backend to be exchanged for an [access token](#access-token) and possibly a [refresh token](#refresh-token).
+
+## Collators (search plugin)
+
+Collators transform streams of [documents](#documents) into searchable texts. They're usually responsible for the data transformation and definition and collection process for specific [documents](#documents). Part of [Backstage Search](#search).
+
+## Component (catalog plugin)
+
+A software product that is managed in the Backstage [Software Catalog](#software-catalog). A component can be a service, website, library, data pipeline, or any other piece of software managed as a single project.
+
+https://backstage.io/docs/features/software-catalog/system-model
+
+## Condition (permission plugin)
+
+Conditions are used to return a conditional decision from a policy. They contain information about a given entity and restrictions on what types of users can view that entity.
+
+## Conditional decisions
+
+[Rules](#permission-rules) need additional data before they can be used in a decision. Once a [rule](#permission-rule) is bound to relevant information it forms a [condition](#condition). Conditional decisions tell the [permission framework](#permission) to delegate evaluation to the [plugin](#plugin) that owns the corresponding [resource](#permission-resource). Permission requests that result in a conditional decision are allowed if all of the provided conditions evaluate to be true.
+
+## Contributor
+
+A volunteer who helps to improve an OSS product such as Backstage. This volunteer effort includes coding, testing, technical writing, user support, and other work. A [user role](#user-role).
+
+## Declarative integration
+
+A new paradigm for Backstage frontend plugins, allowing definition in config files instead of hosting complete React pages.
```

Comment:
```suggestion
A new paradigm for Backstage frontend plugins, allowing definition in config files instead of hosting complete React pages. See [The Frontend System](https://backstage.io/docs/frontend-system).
```

(Suggest using this syntax for all references back to the documentation. Also, it can go at the end of the definition paragraph.)

Reply:
👍 , I'll have to write a doc about writing for the glossary at this rate :)

Reply:
Updated in https://github.com/backstage/backstage/pull/22542/commits/4944f29f2cc61dbd29019cd5693c051301f8497f and added a guide too

Reply:
Nice!